### PR TITLE
[HttpKernel] Tests for DataCollector

### PR DESCRIPTION
**Pull request by vladis84**
| Q             | A
| ------------- | ---
| Branch?       |  5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/DataCollector/TimeDataCollector.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public function collect(Request $request, Response $response, \Throwable $except
 
     public function reset()
     {
-        $this->data = [];
+        $this->data = ['events' => [], 'stopwatch_installed' => false, 'start_time' => 0];
```
Comment (nicolas-grekas) [inline src/Symfony/Component/HttpKernel/DataCollector/TimeDataCollector.php:61]: we should also do this in the constructor then

**Review by nicolas-grekas — COMMENTED** [review]
please have a look at fabbot failures

File: src/Symfony/Component/HttpKernel/DataCollector/TimeDataCollector.php (hunk 1 above)
Comment (vladis84) [reply src/Symfony/Component/HttpKernel/DataCollector/TimeDataCollector.php:61]: fixed

**Comment by nicolas-grekas** [comment]
Thank you @vladis84.

**Pull request merged**